src/Core/src/Handlers/Editor/EditorHandler.iOS.cs

```diff
@@ -19,5 +21,14 @@ public static void MapText(EditorHandler handler, IEditor editor)
 		{
 			handler.TypedNativeView?.UpdateText(editor);
 		}
+
+		public static void MapFont(EditorHandler handler, IEditor editor)
+		{
+			var services = App.Current?.Services ??
+				throw new InvalidOperationException($"Unable to find service provider, the App.Current.Services was null.");
```

Comment:
Not related with this PR, but we have this message repeated in several places. Maybe we can create some specific exceptions like `FontInitializationException` or group all the messages etc.